app/pages/sections/order-items/catalogue-solutions/dashboard/template.test.js

```diff
@@ -139,8 +149,15 @@ describe('Catalogue-solutions - Dashboard page', () => {
               dataTestId: 'orderItem2-catalogueItemName',
             },
             {
-              data: 'Recipient Two (recipient-2)',
-              dataTestId: 'orderItem2-serviceRecipient',
+              data: 'per patient per year',
+              dataTestId: 'orderItem2-unitOfOrder',
+            },
+            {
+              expandableSection: {
+                dataTestId: 'orderItem2-serviceRecipients',
+                title: 'Service recipients (ODS code)',
+                innerComponent: 'Recipient Three (recipient-3',
```

Comment:
Should this have a closing bracket to be more realistic? Also, might it be worth refactoring to a constant that can be reused in the assertion below? (There may be one or two other possibilities for reuse around here as well I think.)